src/CovidSim.cpp

```diff
@@ -5097,29 +5097,34 @@ void RecordSample(double t, int n)
 					if (DesiredAccuracy < 0.05) DesiredAccuracy = 0.05;
 					fprintf(stderr, "\n** %i %lf %lf | %lg / %lg \t", P.ModelCalibIteration, t, P.DateTriggerReached_SimTime + P.DateTriggerReached_CalTime - P.Interventions_StartDate_CalTime, P.HolidaysStartDay_SimTime,RatioPredictedObserved);
 					fprintf(stderr, "| %i %i %i %i -> ", trigAlert, trigAlertCases, P.AlertTriggerAfterIntervThreshold, P.CaseOrDeathThresholdBeforeAlert);
-					if ((P.ModelCalibIteration >= 2) && ((((RatioPredictedObserved - 1.0) <= DesiredAccuracy) && (RatioPredictedObserved >= 1)) || (((1.0 - RatioPredictedObserved) <= DesiredAccuracy) && (RatioPredictedObserved < 1))))
-						P.StopCalibration = 1;
-					else if (P.ModelCalibIteration == 0)
+					if (P.ModelCalibIteration == 0)
 					{
 						// rescale threshold
 						k = (int)(((double)P.CaseOrDeathThresholdBeforeAlert) / RatioPredictedObserved);
```

Comment:
RatioPredictedObserved is double, so this can just be
k = (int) (P.CaseOrDeathThresholdBeforeAlert / RatioPredictedObserved);